spec/lib/query/proposal/search_spec.rb

```diff
@@ -83,7 +83,7 @@
       user = create(:user, client_slug: "test")
 
       proposal1 = create(:proposal, id: 199, requester: user)
-      test_client_request1 = create(:test_client_request, proposal: proposal1)
+      create(:test_client_request, proposal: proposal1)
```

Comment:
unused var. (I have a plugin that highlights these for me)

you think we should re-add and prepend with underscore to show that they are known unused vars?